cli_tools/gce_vm_image_import/importer/args.go

```diff
@@ -86,13 +87,19 @@ func NewImportArguments(args []string) (ImportArguments, error) {
 	flagSet.SetOutput(ioutil.Discard)
 
 	parsed := ImportArguments{
-		ExecutionID: path.RandString(5),
 		WorkflowDir: filepath.Join(filepath.Dir(os.Args[0]), workflowDir),
 	}
 
 	parsed.registerFlags(flagSet)
 
-	return parsed, flagSet.Parse(args)
+	if err := flagSet.Parse(args); err != nil {
+		return parsed, err
+	}
+
+	if parsed.ExecutionID == "" {
+		parsed.ExecutionID = path.RandString(5)
+	}
```

Comment:
This naturally belongs to args.ValidateAndPopulate() or populator.PopulateMissingParameters(). Probably easier to do it in args.ValidateAndPopulate(), similarly to what we do for network.